ui/app/js/controllers/releases_controller.js

```diff
@@ -64,8 +56,8 @@ function($scope, $routeParams, $location, $timeout, Releases, Search, $modal, Pa
   if ($scope.release_name) {
     $scope.ordering_options = [
       {
-        text: "Data Version",
-        value: "-data_version"
+        text: "Timestamp",
+        value: "-timestamp"
```

Comment:
While testing, I ended up doing deletes/inserts of the same Release a bunch of times, and discovered that we end up with duplicate data versions (even with the current code). Sorting by timestamp here improves the situation by actually showing you the changes in the order that they were made, rather than relegating both inserts (which have data_version=1) at the very last page.

Reply:
Very nice! 👍